Library/Formula/msmtp.rb

```diff
@@ -12,6 +12,15 @@ class Msmtp < Formula
 
   depends_on "pkg-config" => :build
   depends_on "openssl"
+  depends_on "gnutls" => :optional
+  depends_on "gsasl" => :optional
+  depends_on "libidn" => :optional
+  depends_on "libsecret" => :optional
+
+  option "with-gsasl", "Use GNU SASL authentication library"
+  option "with-libidn", "Build with support for Internationalized Domain Names"
+  option "with-libsecret"
+  option "with-gnutls", "Use GNU Transport Layer Security library"
```

Comment:
1. It would be useful to have SASL enabled by default. 
2. Not sure about `libidn` but is present in Homebrew.
2. As per `msmtp` docs `gnutls` is preferred TLS method. 
3. `libsecret` is already available in Homebrew, and is useful.